apiv6/generate_client.sh

```diff
@@ -56,33 +92,29 @@ function cleanup_non_needed_files() {
 
     generated_content_dir="$1"
 
+    rm "$generated_content_dir/.travis.yml"
+    rm "$generated_content_dir/git_push.sh"
     rm "$generated_content_dir/go.mod"
     rm "$generated_content_dir/go.sum"
     rm "$generated_content_dir/README.md"
-    rm -rf "$generated_content_dir/docs"
     rm -rf "$generated_content_dir/api"
```

Comment:
If the spec is autodownloaded, please keep the source spec around for reference if something breaks

Reply:
Modified the script not to remove the spec and docs. Docs do not break anything and maybe might be useful for quickly glancing the functionality by library devs. All gitignored.